### PR TITLE
Edit person builder

### PR DESCRIPTION
**Pull request by VedJoshi**
added role and wedding.

### CODE REVIEW

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 1]
```diff
@@ -36,37 +40,52 @@ public PersonBuilder() {
...
+     * Parses the {@code tags} into a {@code Set<Tag>} and sets it to the {@code Person} that we are building.
+     *
+     * @param tags The tags to set.
+     * @return The updated {@code PersonBuilder} instance.
      */
-    public PersonBuilder withTags(String ... tags) {
+    public PersonBuilder withTags(String... tags) {
         this.tags = SampleDataUtil.getTagSet(tags);
         return this;
     }
```
Comment (SherniceSng) [inline src/test/java/seedu/address/testutil/PersonBuilder.java:83]: Maybe can remove withTags method afterwards, if we don't end up using it?

**Review by SherniceSng — COMMENTED** [review]
LGTM.

File: src/main/java/seedu/address/logic/commands/TagCommand.java [hunk 2]
```diff
@@ -62,18 +61,6 @@ public TagCommand(Name name, TagPersonDescriptor tagPersonDescriptor) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        // checks if multiple persons contain same keyword in name
```
Comment (Droas590) [inline src/main/java/seedu/address/logic/commands/TagCommand.java:65]: nice code clean up

**Review by Droas590 — APPROVED** [review]
LGTM

**Pull request merged**